do/parser.js

```diff
@@ -32,6 +32,16 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
     result.mime     = 'HTML';
     result.unitid = param.docid.split('-')[1];
     result.online_identifier = param.docid.split('-')[1];
+  } else if (/^\/audio$/i.test(path)) {
```

Comment:
May be interessing to check if `param.docid` is present (and not null/undefined) to avoid issues.